docs/language/contracts.rst

```diff
@@ -24,15 +24,15 @@ If a constructor has arguments, they must be supplied when the contract is deplo
 If a contract is expected to receive value on instantiation, the constructor should be declared ``payable``.
 
 .. note::
-    On Substrate, constructors have a name. Solang allows naming constructors in the substrate target:
+    On Substrate, constructors may have a name. Solang allows naming constructors in the substrate target:
```

Comment:
These two phrases say exactly the same thing. They are redundant. I suggest we eliminate one of them.

```suggestion
    Solang allows naming constructors in the Substrate target:
```